compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/platform/PlatformTextInputSession.skiko.kt

```diff
@@ -17,10 +17,15 @@
 package androidx.compose.ui.platform
 
 import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.ui.geometry.Rect
 import androidx.compose.ui.text.input.TextFieldValue
 
 actual interface PlatformTextInputSession {
     actual suspend fun startInputMethod(request: PlatformTextInputMethodRequest): Nothing
+
     @ExperimentalComposeUiApi
     fun updateSelectionState(newState: TextFieldValue) = Unit
+
+    @ExperimentalComposeUiApi
```

Comment:
We should stop using `ExperimentalComposeUiApi` for cross-module communication. What's the plan to make this part stable?

Reply:
I don't know, but it sounds like a global question, and I'd rather not solve it in the scope of this PR.

Reply:
Okay, let's postpone this discussion for now